registry-analysis/Readme.md

```diff
@@ -22,11 +22,112 @@ Examples:
 
 Volunteer: Jan Romann
 
-Examples:
-- https://w3c.github.io/tt-profile-registry/#registry
-- https://www.w3.org/TR/mse-byte-stream-format-registry/
-- https://www.w3.org/TR/did-spec-registries/
-- https://www.w3.org/2005/04/xpointer-policy
+#### TTML Media Type Definition and Profile Registry
+
+This [document](https://w3c.github.io/tt-profile-registry/#registry) defines a specific media type and a registry for associated "processor profiles".
+The registry, its entries and the modification process are specified as follows:
+
+- Entries must contain a unique ID, an absolute URI that identifies the profile, and contact information of the registrant
+- Entries *should* contain links to the specification of the profile itself and its corresponding vocabulary
+- Entries can be added, changed, or deleted
+  - Although deletion is possible, entries are intended to be stable
+- Requests are made via an email to the WG chair, will be decided on in a WG meeting
+- Decisions are made via consensus, requestor is notified accordingly
+
+#### Media Source Extensions Byte Stream Format Registry
+
+This [document](https://www.w3.org/TR/mse-byte-stream-format-registry/) defines
+a registry that maps between MIME-type/subtype pairs and byte stream format
+specifications.
+These are relevant for
+[`SourceBuffer` objects](https://www.w3.org/TR/media-source/#sourcebuffer) which
+are used for cosuming audio and video streams.
+
+The registry, its entries and the modification process are specified as follows:
+
+- Each entry consists of MIME type/subtype which is mapped to a publicly available specification for a byte stream format. Furthermore, it has to be specified if a flag for generating timestamps needs to be set with the byte stream format.
+- It is recommended that the specification should be available without fees.
+- All requirements mentioned in the [Media Source Extensions](https://www.w3.org/TR/media-source/#byte-stream-formats) specification must be fulfilled.
```

Comment:
This is a good point for us, meaning we can specify the requirements of bindings in general in one place and registry requirements point to that.